buildspec.yml

```diff
@@ -14,7 +14,7 @@ phases:
       - 'echo "Run jscs..."'
       - './node_modules/.bin/gulp jscs'
       - 'echo "Package as ZIP file"'
-      - 'zip -r "suitec-preview-service.zip" * -x logs\*'
+      - 'zip -r "suitec-preview-service.zip" * .e* .r*'
```

Comment:
Good catch, @paulkerschen  Typo. I'm going to amend to: `*  .*`. I see no problem having `.git*` files included.